# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/chrome-webidl/ChannelWrapper.webidl
# Commit: f2df73fda468
# Full Hash: f2df73fda46859150605f33bcdaeb49ab7037720
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-10-19 21:40:34
# Regressor Bug: 1588899
# File Overlap Count: 1
# Description:
#   Bug 1588899 - P1. Move classification flags related method to nsIClassifiedChannel. r=Ehsan,baku
#   
#   This is where it should have been in the first place. Those attributes belong there.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D49577
# ==============================================================================

diff -r b98968eb71d3 -r f2df73fda468 dom/chrome-webidl/ChannelWrapper.webidl
--- a/dom/chrome-webidl/ChannelWrapper.webidl	Thu Oct 17 15:24:31 2019 +0000
+++ b/dom/chrome-webidl/ChannelWrapper.webidl	Sat Oct 19 04:30:24 2019 +0000
@@ -38,7 +38,7 @@
 };
 
 /**
- * String versions of CLASSIFIED_* tracking flags from nsHttpChannel.idl
+ * String versions of CLASSIFIED_* tracking flags from nsIClassifiedChannel.idl
  */
 enum MozUrlClassificationFlags {
   "fingerprinting",